queries/javascript/injections.scm

```diff
@@ -9,6 +9,6 @@
 
 (call_expression
   function: ((identifier) @_name
-             (#eq? @_name "gql"))
+             (#match? @_name "g(raph)?ql"))
```

Comment:
Maybe this more readable as `^(gql|graphql)$`

Reply:
Also, isn't this case already covered with the query above?

Reply:
doesn't seem to work for me, I didn't get any injection without adding this

Reply:
Does it work if you put this query first and the more general below this one?

Reply:
wait, hold on, I think I got it working now. Not sure why it didn't work previously, sorry for the noise :laughing: